packages/authentication/src/definitions.ts

```diff
@@ -483,6 +483,17 @@ export interface FirebaseAuthenticationPlugin {
     eventName: 'authStateChange',
     listenerFunc: AuthStateChangeListener,
   ): Promise<PluginListenerHandle>;
+  /**
+   * Listen for changes to the signed-in user's ID token changes.
+   *
+   * **Attention:** This listener will not be triggered automatically upon ID token expiration.
+   *
```

Comment:
```suggestion

```

For plugins that integrate external SDKs, I therefore always refer to the documentation of the external SDK for questions and do not copy anything into the plugin, as the documentation can be outdated very quickly.